src/game/lara1gun.c

```diff
@@ -164,27 +164,22 @@ void AnimateShotgun()
             }
         }
     } else {
-        if (ani == AF_SG_AIM && Input.action) {
-            ani++;
-        } else if (ani > AF_SG_AIM && ani < AF_SG_DRAW) {
-            ani++;
-            if (ani == AF_SG_DRAW) {
-                ani = Input.action ? AF_SG_RECOIL : AF_SG_UNAIM;
-            }
-        } else {
-            if (ani == AF_SG_RECOIL) {
+        if (T1MConfig.enable_shotgun_target_fix && Input.action
```

Comment:
Do we want the `Input.Action` check here? The original checks `Input.Action` after this, not always for true.

Reply:
I tried to make it kind of match the pistols in lara2gun.c AnimatePistols.
`if (Lara.right_arm.lock || (Input.action && !Lara.target))`